site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -223,12 +230,26 @@ The issuer used by service providers (i.e. Google, Zendesk, etc.) to identify th
 [field]#Audience# [optional]#Optional#::
 Some service providers require a different audience (such as Zendesk). You can leave this blank if the audience is the same as the issuer.
 
-[field]#Callback URL (ACS)# [required]#Required#::
-The URL that FusionAuth returns the user to after they have logged in via SAML v2, also known as the Assertion Consumer Service URL (ACS).
+[field]#Authorized redirect URLs# [required]#Required#::
+One or more allowed URLs that FusionAuth may redirect to after the user has logged in via SAML v2, also known as the Assertion Consumer Service URL (ACS).
 
 [field]#Logout URL# [optional]#Optional#::
 The URL that the user is redirected to after they are logged out. Usually this is the starting location of the application.
 
+[field]#Debug enabled# [optional]#Optional#::
+Enable debug to create an event log to assist you in debugging integration errors.
+
+[field]#Require signature# [optional]#Optional#::
+When enabled, all unsigned requests will be rejected.
+
+[field]#Default verification key# [optional]#Optional# [since]#Available since 1.20.0#::
+The default key used to verify the signature in the following scenarios:
+* When the public key cannot be determined by the `KeyInfo` element when using POST bindings.
+* When using HTTP Redirect bindings.
+
++
```

Comment:
Extra `+`?

Reply:
Originally, the plus dropped the indent back. Now removed in favor of the text from the API.